test/python/test_util.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring,no-member
+
+# Copyright 2017 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from qiskit._util import _check_ibmqe_version, API_NAME
+from .common import QiskitTestCase
+
+
+class TestUtil(QiskitTestCase):
+    """Tests for qiskit/_util.py"""
+
+    def test_check_ibmqe_version(self):
+        """Requiered IBMQE version."""
+        with self.assertNoLogs('qiskit._util', level='WARNING'):
+            _check_ibmqe_version()
+
+    def test_api_name(self):
```

Comment:
Hmm, can you check if this test is really meaningful, as it seems to just compare the value of a variable that is hard-coded on the utils?

Reply:
I tested by downgrading the lib. So it worked and it was meaningful. I retested now, with the proper check in utils. Still working.